### ISSUE TITLE
Running RC commands via NPC crashes server via watchdog.

### ISSUE THREAD

**Issue by TheDeafCreeper** [Aelysium-Group/rusty-connector#8]
Full Logs: https://pastebin.com/D0bc2mYr

Also Citizens wiki states By default, "/npc command" will run commands as the server.

Single Stacktrace:
```
java.base@18.0.2-ea/jdk.internal.misc.Unsafe.park(Native Method)
[21:31:59 ERROR]:               java.base@18.0.2-ea/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
[21:31:59 ERROR]:               java.base@18.0.2-ea/java.util.concurrent.CompletableFuture$Signaller.block(CompletableFuture.java:1864)
[21:31:59 ERROR]:               java.base@18.0.2-ea/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3464)
[21:31:59 ERROR]:               java.base@18.0.2-ea/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3435)
[21:31:59 ERROR]:               java.base@18.0.2-ea/java.util.concurrent.CompletableFuture.waitingGet(CompletableFuture.java:1898)
[21:31:59 ERROR]:               java.base@18.0.2-ea/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2072)
[21:31:59 ERROR]:               RustyConnector.jar//io.lettuce.core.DefaultConnectionFuture.get(DefaultConnectionFuture.java:69)
[21:31:59 ERROR]:               RustyConnector.jar//io.lettuce.core.AbstractRedisClient.getConnection(AbstractRedisClient.java:345)
[21:31:59 ERROR]:               RustyConnector.jar//io.lettuce.core.RedisClient.connectPubSub(RedisClient.java:336)
[21:31:59 ERROR]:               RustyConnector.jar//group.aelysium.rustyconnector.core.lib.database.redis.RedisPublisher.publish(RedisPublisher.java:27)
[21:31:59 ERROR]:               RustyConnector.jar//group.aelysium.rustyconnector.plugin.paper.lib.VirtualServerProcessor.sendToOtherFamily(VirtualServerProcessor.java:179)
[21:31:59 ERROR]:               RustyConnector.jar//group.aelysium.rustyconnector.plugin.paper.commands.CommandRusty.lambda$create$2(CommandRusty.java:55)
[21:31:59 ERROR]:               RustyConnector.jar//group.aelysium.rustyconnector.plugin.paper.commands.CommandRusty$$Lambda$9879/0x000000080294b808.accept(Unknown Source)
[21:31:59 ERROR]:               RustyConnector.jar//cloud.commandframework.bukkit.BukkitSynchronizer.lambda$runSynchronous$0(BukkitSynchronizer.java:53)
[21:31:59 ERROR]:               RustyConnector.jar//cloud.commandframework.bukkit.BukkitSynchronizer$$Lambda$9885/0x0000000802950630.run(Unknown Source)
[21:31:59 ERROR]:               org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
[21:31:59 ERROR]:               org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483)
[21:31:59 ERROR]:               net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1498)
[21:31:59 ERROR]:               net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:447)
[21:31:59 ERROR]:               net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1412)
[21:31:59 ERROR]:               net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1189)
[21:31:59 ERROR]:               net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
[21:31:59 ERROR]:               net.minecraft.server.MinecraftServer$$Lambda$4511/0x00000008015bd6e0.run(Unknown Source)
[21:31:59 ERROR]:               java.base@18.0.2-ea/java.lang.Thread.run(Thread.java:833)
```

**Comment by nathan-i-martin** [comment]
My intuition says this has something to do with either #11 or the command sender not being correct in some way.
I doubt it's the latter but I'm going to look into that as a possibility anyways.

**Comment by nathan-i-martin** [comment]
@TheDeafCreeper can you try testing this again using the criteria laid out in #11?

**Comment by TheDeafCreeper** [comment]
NPCs seem to be fixed, at least in 0.5.2 beta.

**Comment by nathan-i-martin** [comment]
Great, I think this may have been caused by whatever issue was causing #11. Some sort of weirdness causes all crap to hit the fan and seems to crash the server.
